utils/important_models.txt

```diff
@@ -0,0 +1,4 @@
+models/llama
+models/mistral
+models/mixtral
+models/gemma
```

Comment:
```suggestion
models/gemma
models/whisper
```
let's at least add this one, and cc @amyeroberts and @NielsRogge maybe Llava?

Reply:
Yep - llava is good to add!